lib/matplotlib/tests/test_dates.py

```diff
@@ -163,11 +159,19 @@ def test_too_many_date_ticks(caplog):
     assert len(caplog.records) > 0
 
 
+def _new_epoch_decorator(thefunc):
+    def wrapper():
```

Comment:
Do you need to use `functools.wraps` here to make sure the tests actually get run?

Reply:
Not sure I follow?   The tests are running as far as I can tell...

Reply:
If you put `assert False` in them do they fail as expected?  

If you ask this function what it's name it is will tell you `wrapper` (not the name of the decorated function).  I'm not sure off the top of my head how much introspection of that pytest uses as part of it's test discovery.

Reply:
I tested locally and pytest is not as magical as I was concerned.